Exiled.API/Features/Scp914.cs

```diff
@@ -74,20 +83,52 @@ public static Scp914Mode ConfigMode
         /// </summary>
         public static Transform OutputBooth => Scp914Controller.OutputChamber;
 
+        /// <summary>
+        /// All the Object casted inside 914.
+        /// </summary>
+        /// <param name="playersret">The <see cref="List{Player}"/> to return.</param>
+        /// <param name="pickupsret">The <see cref="List{Pickup}"/> to return.</param>
+        /// <returns>All GameObject inside Scp914.</returns>
+        public static IEnumerable<GameObject> Scp914InputObject(out IEnumerable<Player> playersret, out IEnumerable<Pickup> pickupsret)
```

Comment:
Perhaps the returns should be named "players" and "pickups", as opposed to "playersret" and "pickupsret"